integration/tests/basic/evm/test_solana_interoperability.py

```diff
@@ -156,6 +156,26 @@ def test_counter_execute(self, call_solana_caller, counter_resource_address: byt
         event_logs = call_solana_caller.events.LogBytes().process_receipt(resp)
         assert int.from_bytes(event_logs[0].args.value, byteorder="little") == next(get_counter_value)
 
+    def test_counter_execute_overload(self, call_solana_caller, counter_resource_address: bytes, get_counter_value):
```

Comment:
let's rename tests in something like "without lamports" because difference is only in it
"overload" is just a way to get similar functions

Reply:
or we can parametrise exist test as the differens is only in one single parameter not to have the same code twice

Reply:
test_counter_execute(...., lamports=None or uint)